client/components/global-notices/index.jsx

```diff
@@ -12,11 +12,13 @@ import Notice from 'components/notice';
 import NoticeAction from 'components/notice/notice-action';
 import notices from 'notices';
 import observe from 'lib/mixins/data-observe';
-import DeleteSiteNotices from './delete-site-notices';
+import DeleteSiteNotices from 'notices/delete-site-notices';
+import { connect } from 'react-redux';
+import { noticesMapDispatchToProps } from 'state/ui/notices/actions'
 
 const debug = debugModule( 'calypso:notices' );
 
-export default React.createClass( {
+const NoticesList =  React.createClass( {
```

Comment:
Additional space after the `=` to be removed.

Consider using an ESLint plugin for your preferred editor if you're not already, as they often make these errors easy to spot.

![image](https://cloud.githubusercontent.com/assets/1779930/11815618/7fe9272a-a31a-11e5-97b6-4244bdf14778.png)